include/xtensor/xstrided_view_base.hpp

```diff
@@ -75,6 +76,52 @@ namespace xt
             size_type m_size;
         };
 
+        template <class CT, layout_type L>
+        class linear_flat_expression_adaptor : public flat_expression_adaptor<CT, L>
```

Comment:
Why inheriting from `flat_expression_adaptor` here? It does not seem that you reuse anything from it.

Reply:
@JohanMabille  I only implemented the linear iterator methods for the new class. `linear_flat_expression_adaptor` relies on inheriting the implementation of `flat_expression_adaptor` for calls made to methods such as `cbegin`.

Reply:
@JohanMabille what do you think? Should I implement this differently?